UPGRADE-5.0.md

```diff
@@ -219,6 +219,12 @@ Process
    $process = Process::fromShellCommandline('ls -l');
    ```
 
+Routing
+-------
+
+ * `Route` and `CompiledRoute` don't implement `Serializable` anymore; if you serialize
+   them, please ensure your unserialization logic is resilient to this change
```

Comment:
`resilient` is not a basic English word; it's better to use something easier to understand. Also, we may need a bit more info about what should developers do in the unserialization phase if they serialize them.

Reply:
updated